### ISSUE TITLE
Screen Reader doesn't announce a Property Grid panel state - expanded or collapsed

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-creator#3679]
The screen reader should announce the state of a property grid panel - expanded or collapsed.
Original issue:
[T11295 - Accessibility issue : Incorrect role is defined for 'General' control in designer page](https://surveyjs.answerdesk.io/internal/ticket/details/T11295)

**Comment by JaneSjs** [comment]
The NVDA reader doesn't announce the Collapsed state:
![Uploading image.png…]()

**Comment by dmitrykurmanov** [comment]
> The NVDA reader doesn't announce the Collapsed state: ![2023-01-20_18-23-43](https://user-images.githubusercontent.com/22372972/213735734-7ad1bd5b-7e79-4b65-a87a-97e99153809c.png)

Knockout only issue
